django_short_urls/auth.py

```diff
@@ -0,0 +1,36 @@
+# coding=utf-8
+
+"""Module to assist with authentication"""
+
+from __future__ import unicode_literals
+
+from decorator import decorator
+
+from django_short_urls.models import User
+from http.status import HTTP_UNAUTHORIZED
+from http.utils import response
+
+
+@decorator
+def login_with_basic_auth_required(view, request, *args, **kwargs):
```

Comment:
Nevermind, you already covered that previously with the RequestFactory in `tests/new_test.py`